Resources/Locale/en-US/chat/highlights.ftl

```diff
@@ -0,0 +1,57 @@
+# Command
+highlights-captain = Captain, "Cap", Command
+highlights-head-of-personnel = Head Of Personnel, "HoP", Service, Command
+highlights-chief-engineer = Chief Engineer, "CE", Engineering, Engineer, "Engi", Command
+highlights-chief-medical-officer = Chief Medical Officer, "CMO", MedBay, "Med", Command
+highlights-head-of-security = Head of Security, "HoS", Security, "Sec", Command
+highlights-quartermaster = Quartermaster, "QM", Cargo, Command
+highlights-research-director = Research Director, "RD", Science, "Sci", Command
+
+# Security
+highlights-detective = Detective, "Det", Security, "Sec"
+highlights-security-cadet = Security Cadet, Cadet, Security, "Sec"
+highlights-security-officer = Security Officer, Secoff, Officer, Security, "Sec"
+highlights-warden = Warden, "Ward", Security, "Sec"
+
+# Cargo
+highlights-cargo-technician = Cargo Technician, Cargo Tech, Cargo
+highlights-salvage-specialist = Salvage Specialist, Salvager, Salvage, "Salv", Miner
+
+# Engineering
+highlights-atmospheric-technician = Atmospheric Technician, Atmos tech, Atmospheric, Atmos
+highlights-station-engineer = Station Engineer, Engineering, Engineer, "Engi"
+highlights-technical-assistant = Technical Assistant, Tech Assistant, Engineering, Engineer, "Engi"
+
+# Medical
+highlights-chemist = Chemist, Chemistry, MedBay
```

Comment:
```suggestion
# Medical
highlights-chemist = Chemist, Chemistry, "Chem", MedBay, "Med"
```

I see chemists called "Chem" often so I think this would work